### DISCUSSION TITLE
怎么使用Block引用 的方式，在外部引用进行批注。

### DISCUSSION THREAD

**Discussion by 0xVote** [zhaohongxuan/obsidian-weread-plugin#146] [General]
本插件是覆盖式更新，请不要在同步的文件里修改内容，写永久笔记（为什么写永久笔记参考《卡片笔记写作法》）的时候可以使用Block引用 的方式，在外部引用进行批注。

**Comment by 0xVote** [comment]
类似wiki链接汇聚一点，怎么写笔记啊？作者能不能出一个教程或者哪位小伙伴会的分享一下![image](https://user-images.githubusercontent.com/58676129/225479220-a82c76ba-2a26-433a-9de1-360b10787c68.jpeg)

**Reply by YutingGu** [reply]
如下写两个方括号就可以定位到具体file，然后去掉 | 之后的内容，改成^就可以直接引用具体的小节并创建链接了
<img width="1469" alt="Screenshot 2023-08-12 at 23 11 36" src="https://github.com/zhaohongxuan/obsidian-weread-plugin/assets/43612429/62657de6-9b21-4eaa-bd5e-fe7cb490e894">
